api-catalog-ui/frontend/cypress/e2e/graphql/graphql-apiml.cy.js

```diff
@@ -37,7 +37,7 @@ const PATH_TO_PLAYGROUND_INPUT_DATA =
     '#graphiql-session > div:nth-child(1) > div > div:nth-child(1) > section > div.graphiql-editor > div > div.CodeMirror-scroll > div.CodeMirror-sizer > div > div > div > div.CodeMirror-code';
 
 function login() {
-    cy.visit(`${Cypress.env('catalogHomePage')}/#/login`);
+    cy.visit(`${Cypress.env('catalogHomePage')}/index.html#/login`);
```

Comment:
why does it require index.html now?

Reply:
It basically doesn't because there is a redirect, but Cypress didn't process it well. The root cause is that meaning of / in the url was changed. The code in Catalog solve it by 3xx response code. If there is the original URL it receive the redirect (correct one) but the test fails.